cryptus-app/components/market_overview/net_worth/time_interval/time_interval.module.scss

```diff
@@ -0,0 +1,42 @@
+@use "/styles/variables" as v;
+
+.icon {
+  position: relative;
+  background-color: #ffffff;
```

Comment:
use global

Reply:
What and where are globals ? @alexandrelafleur
can't seem to find .container in globals.css nor _variabeles.scss